Automattic/wp-calypso
# Starter Page Templates: fix preview style override specificity

The `.editor-styles-wrapper` gets applied to the `.block-editor-block-preview__container` element, not to one of its children, thus style overrides in the template picker's preview are not being picked up. Along with a few previous overrides, this corrects the negative margins of `align:full` blocks and fixes #38884 

<img width="1194" alt="Screen Shot 2020-01-21 at 1 08 18 PM" src="https://user-images.githubusercontent.com/942359/72853410-4f780400-3c7f-11ea-8b2c-90acb4ea7733.png">

**To test:**
- create a new page on a site with a template-first theme
- select a page template with an `align:full` block (i.e. Barnsbury, Menu, Contact)
- verify that the preview does not overflow the preview container on the left or right
